cardano-crypto-praos/src/Cardano/Crypto/VRF/PraosBatchCompat.hs

```diff
@@ -256,8 +255,8 @@ copyFromByteString ptr bs lenExpected =
 
 seedFromBytes :: ByteString -> Seed
```

Comment:
No `HasCallStack`?  For any reason in particular?  out of curiosity

Reply:
Cause there is only one use case in this module ;)
It was previously wrongfully exported, so I actually did add `HasCallStack` :smile: but then once I removed it from exports I removed the `HasCallStack` as well.